documentation/source/install_devel.rst

```diff
@@ -4,12 +4,20 @@
 Installing the Development Version
 **********************************
 
-To install the development version of cocotb, run:
+To install the development version of cocotb on Windows, run the following:
+
+.. code-block:: bash
+
+    pip install --global-option build_ext --global-option --compiler=mingw32 https://github.com/cocotb/cocotb/archive/master.zip
```

Comment:
Same for `pip install cocotb` ;-(

Reply:
Restructured the release installation part and added it there too.